[2025-05-12T16:25:32.173+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql manual__2025-05-12T16:25:28+00:00 [queued]>
[2025-05-12T16:25:32.181+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql manual__2025-05-12T16:25:28+00:00 [queued]>
[2025-05-12T16:25:32.181+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-12T16:25:32.194+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): sync_and_load_mysql> on 2025-05-12 16:25:28+00:00
[2025-05-12T16:25:32.199+0000] {standard_task_runner.py:60} INFO - Started process 1214 to run task
[2025-05-12T16:25:32.202+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mymind_mongo_to_mysql_etl', 'sync_and_load_mysql', 'manual__2025-05-12T16:25:28+00:00', '--job-id', '35987', '--raw', '--subdir', 'DAGS_FOLDER/mymind_etl_dag.py', '--cfg-path', '/tmp/tmphy3uboyw']
[2025-05-12T16:25:32.203+0000] {standard_task_runner.py:88} INFO - Job 35987: Subtask sync_and_load_mysql
[2025-05-12T16:25:32.247+0000] {task_command.py:423} INFO - Running <TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql manual__2025-05-12T16:25:28+00:00 [running]> on host 05e28894127a
[2025-05-12T16:25:32.333+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mymind_mongo_to_mysql_etl' AIRFLOW_CTX_TASK_ID='sync_and_load_mysql' AIRFLOW_CTX_EXECUTION_DATE='2025-05-12T16:25:28+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-12T16:25:28+00:00'
[2025-05-12T16:25:32.334+0000] {mymind_etl_dag.py:302} INFO - Iniciando sincronización y carga a MySQL: 7 usuarios, 20 transcripciones.
[2025-05-12T16:25:32.342+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-05-12T16:25:32.350+0000] {mymind_etl_dag.py:311} INFO - Obteniendo IDs actuales desde MySQL...
[2025-05-12T16:25:32.351+0000] {mymind_etl_dag.py:322} INFO - Usuarios a eliminar: 0
[2025-05-12T16:25:32.351+0000] {mymind_etl_dag.py:323} INFO - Transcripciones a eliminar: 7
[2025-05-12T16:25:32.352+0000] {mymind_etl_dag.py:333} INFO - Se eliminaron 7 transcripciones obsoletas.
[2025-05-12T16:25:32.352+0000] {mymind_etl_dag.py:356} INFO - Cargando 7 usuarios en la tabla users...
[2025-05-12T16:25:32.353+0000] {mymind_etl_dag.py:381} INFO - Usuarios cargados exitosamente.
[2025-05-12T16:25:32.354+0000] {mymind_etl_dag.py:394} INFO - Cargando 20 transcripciones en la tabla transcriptions...
[2025-05-12T16:25:32.356+0000] {mymind_etl_dag.py:424} INFO - Transcripciones cargadas exitosamente.
[2025-05-12T16:25:32.357+0000] {mymind_etl_dag.py:431} INFO - Sincronización y carga a MySQL completada con éxito.
[2025-05-12T16:25:32.357+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-12T16:25:32.366+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mymind_mongo_to_mysql_etl, task_id=sync_and_load_mysql, execution_date=20250512T162528, start_date=20250512T162532, end_date=20250512T162532
[2025-05-12T16:25:32.414+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-12T16:25:32.437+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
